src/lib.rs

```diff
@@ -2488,14 +2469,8 @@ impl<'a> Structure<'a> {
                     const _: () = { #generated };
                 })
             } else {
-                let dummy_const: Ident = sanitize_ident(&format!(
```

Comment:
WE already have a config (`self.underscore_const`) which implements this feature. We should remove that config and make the existing setter method a no-op, simplifying the code.

Reply:
Huh, I was tunnel visioning very hard on this one sorry 😅